internal/service/ipam/model_ipamsvc_range.go

```diff
@@ -0,0 +1,280 @@
+		},
+		MarkdownDescription: "The description for the range. May contain 0 to 1024 characters. Can include UTF-8.",
+	},
+	"created_at": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "Time when the object has been created.",
+	},
+	"dhcp_host": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "The resource identifier.",
+	},
+	"dhcp_options": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: IpamsvcOptionItemResourceSchemaAttributes,
+		},
+		Optional:            true,
+		MarkdownDescription: "The list of DHCP options. May be either a specific option or a group of options.",
+	},
+	"disable_dhcp": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: "Optional. _true_ to disable object. A disabled object is effectively non-existent when generating configuration.  Defaults to _false_.",
+	},
+	"end": schema.StringAttribute{
+		Required: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
+		},
+		MarkdownDescription: "The end IP address of the range.",
+	},
+	"exclusion_ranges": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: IpamsvcExclusionRangeResourceSchemaAttributes,
+		},
+		Optional:            true,
+		MarkdownDescription: "The list of all exclusion ranges in the scope of the range.",
+	},
+	"filters": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: IpamsvcAccessFilterResourceSchemaAttributes,
+		},
+		Optional:            true,
+		MarkdownDescription: "The list of all allow/deny filters of the range.",
+	},
+	"id": schema.StringAttribute{
+		Computed: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.UseStateForUnknown(),
+		},
+		MarkdownDescription: "The resource identifier.",
+	},
+	"inheritance_assigned_hosts": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: InheritanceAssignedHostResourceSchemaAttributes,
+		},
+		Computed:            true,
+		MarkdownDescription: "The list of the inheritance assigned hosts of the object.",
+	},
+	"inheritance_parent": schema.StringAttribute{
+		MarkdownDescription: "The resource identifier.",
+		Computed:            true,
+	},
+	"inheritance_sources": schema.SingleNestedAttribute{
+		Attributes: IpamsvcDHCPOptionsInheritanceResourceSchemaAttributes,
+		Optional:   true,
+		Computed:   true,
+	},
+	"name": schema.StringAttribute{
+		Optional: true,
+		Validators: []validator.String{
+			stringvalidator.LengthBetween(1, 256),
+		},
+		MarkdownDescription: "The name of the range. May contain 1 to 256 characters. Can include UTF-8.",
+	},
+	"parent": schema.StringAttribute{
+		Optional:            true,
+		Computed:            true,
```

Comment:
should be read only?

Reply:
The parent is computed.

Reply:
Yes, so optional:true should be removed right?